code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -728,6 +749,17 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		M.adjustFireLoss(-2)
 	return ..()
 
+/datum/reagent/consumable/ethanol/manly_dorf/ancient
+	name = "The Manliest Dorf"
+	description = "Secret recipe held by dwarves. They sold the old one so they had to make one for armok's sake!"
+	boozepwr = 150
+	glass_name = "The Manliest Dorf"
+
+/datum/reagent/consumable/ethanol/manly_dorf/ancient/on_mob_life(mob/living/carbon/M)
+	if(dorf_mode)
+		M.adjustBruteLoss(-M.drunkenness/50) //Manliest of dwarves
```

Comment:
How high is this while you can still function?

Reply:
not much it is 4 healing at the bachuus levels of drunkness